src/schema/notifications.ts

```diff
@@ -302,8 +322,12 @@ export class ZimbraNotifications {
 		if (items) {
 			items.forEach((i: any) => {
 				const item = normalizeFolder(i);
+				let itemId: any;
```

Comment:
There was another PR - https://github.com/Zimbra/zm-api-js-client/pull/270. Aren't we proceeding with that ?

Reply:
The approch was changed, incorporated the same.